content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -152,14 +152,29 @@ The following table shows which referential action each database supports.
 
 | Database    | Cascade | Restrict | NoAction | SetNull | SetDefault |
 | :---------- | :------ | :------- | :------- | :------ | :--------- |
-| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
+| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**⌘ | **✔️**     |
 | MySQL       | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**†    |
```

Comment:
Maybe to express that this does not actually _work_ anywhere, but is accepted in some cases:
```suggestion
| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | ❌(✔️⌘) | **✔️**     |
| MySQL       | **✔️**  | **✔️**   | **✔️**   | **✔️**  | ❌ (✔️†)    |
```

Reply:
Ok

Reply:
True, so my suggested change only makes sense for MySQL:
```suggestion
| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**⌘ | **✔️**     |
| MySQL       | **✔️**  | **✔️**   | **✔️**   | **✔️**  | ❌ (✔️†)    
```